white-test-app-frontend/src/app/services/http-service.service.ts

```diff
@@ -45,7 +45,7 @@ export class HttpService {
 
   putWithoutBody<T>(endpoint: string, httpOptions = {}): Observable<T> {
     this.setAuthHeader();
-    return this.http.put<T>(endpoint, {
+    return this.http.put<T>(endpoint, null, {
```

Comment:
czyli jednak null? mogłes skorzystać tez ze zwyklego put-a i przekazać to jako body, ale ok